execution/evm/abi/core.go

```diff
@@ -27,10 +27,10 @@ func init() {
 
 var RevertAbi *AbiSpec
 
-func ReadAbiFormulateCallFile(abiLocation, binPath, funcName string, args []string) ([]byte, error) {
+func ReadAbiFormulateCallFile(abiLocation, binPath, funcName string, args []string) ([]byte, bool, error) {
```

Comment:
doc comment

Reply:
Add godoc and give function sensible name